omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -5,577 +5,727 @@
           formVersion="3.0"
           formAddMetadata="yes"
           formUILocation="patientDashboard.visitActions"
-          formOrder="2"
+          formOrder="0"
```

Comment:
@slubwama Global - indent the Javascript and tags to improve readability

Reply:
done